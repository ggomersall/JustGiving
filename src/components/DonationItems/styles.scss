.donation-items__wrapper {
  display: flex;
}

.donation-item {
  width: 100%;
  margin-right: 15px;
}