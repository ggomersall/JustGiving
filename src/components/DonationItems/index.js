import React from 'react';
import DonationItem from '../DonationItem';

import './styles.scss';

const DonationItems = ({donations}) => {
  return(
    <div className="donation-items__wrapper">
      {
        donations && donations.donations.map((donation, i) => {
          return (
            <div className="donation-item" key={i}>
              <DonationItem data={donation} />
            </div>
          )
        })
      }
    </div>
  )
}

export default DonationItems;