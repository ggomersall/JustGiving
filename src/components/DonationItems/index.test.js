import React from 'react';
import { shallow } from 'enzyme';
import DonationItems from './index.js';

describe('Testing DonationItems Component', () => {

  let wrapper;

  test('it should render data correctly', () => {
    const donations = { 
      donations: [
        {donation: 'some data'}, {donation: 'more data'}
      ]
    };
    wrapper = shallow(<DonationItems donations={donations} />);
  });

  test('Snapshot', () => expect(<DonationItems />).toMatchSnapshot());

});

