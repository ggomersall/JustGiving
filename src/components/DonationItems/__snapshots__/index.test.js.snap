// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DonationItems Component Snapshot 1`] = `<DonationItems />`;
